src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts

```diff
@@ -504,13 +511,18 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 		this._model.updateActiveCell(this.cellModel);
 	}
 
-	private findPathRelativeToContent(elementContent: string): string {
+	private findPathRelativeToContent(absolutePathURI: URI): string {
```

Comment:
It would be great to add tests for this logic. It might be easiest to move this.notebookUri to be a param instead and then just make this a file level function